src/StorageSync/StorageSync/RegisteredServer/RegisterServerCommand.cs

```diff
@@ -197,7 +228,9 @@ private RegisteredServer CreateRegisteredResourceInCloud(string resourceGroupNam
                 ClusterId = serverRegistrationData.ClusterId.ToString(),
                 ClusterName = serverRegistrationData.ClusterName,
                 AgentVersion = serverRegistrationData.AgentVersion,
-                ServerCertificate = Convert.ToBase64String(serverRegistrationData.ServerCertificate),
+                //ApplicationId = serverRegistrationData.ApplicationId.HasValue ? serverRegistrationData.ApplicationId.Value.ToString() : null,
```

Comment:
why commented out and null hardcoded?

Reply:
in V19, We dont allow Server registration with application id